oeps/oep-0028-content-theming-in-react.rst

```diff
@@ -0,0 +1,246 @@
+==================================
+OEP-0028: Content Theming in React
+==================================
+
++-----------------+----------------------------------------------------------------+
+| OEP             | :doc:`OEP-0028 </oeps/oep-0028-arch-content-theming-react>`    |
+|                 |                                                                |
+|                 |                                                                |
+|                 |                                                                |
+|                 |                                                                |
++-----------------+----------------------------------------------------------------+
+| Title           | Content Theming in React                                       |
++-----------------+----------------------------------------------------------------+
+| Last Modified   | 2018-08-24                                                     |
++-----------------+----------------------------------------------------------------+
+| Authors         | Braden MacDonald <braden@opencraft.com>,                       |
+|                 | Taranjeet Singh <taranjeet@opencraft.com>                      |
++-----------------+----------------------------------------------------------------+
+| Arbiter         | Ari Rizzitano <arizzitano@edx.org>                             |
++-----------------+----------------------------------------------------------------+
+| Status          | Draft                                                          |
++-----------------+----------------------------------------------------------------+
+| Type            | Architecture                                                   |
++-----------------+----------------------------------------------------------------+
+| Created         | 2018-08-17                                                     |
++-----------------+----------------------------------------------------------------+
+| `Review Period` | 2018-08-17 - 2018-09-02                                        |
++-----------------+----------------------------------------------------------------+
+
+Abstract
+-------
+
+OEP-0023 [1] addresses the styling part of a theming system based on React. This proposal adds guidelines for content modification aspect to the React based theming system
```

Comment:
Nit: "This proposal adds guidelines for _content_ modification ~~aspect~~ to the React-based theming system."

(remove "aspect", add period)